Description: this patch fixes a spelling error in arc's man page.
             This error is already fixed upstream but not released yet.
Author: Klaus Reimer <kay@debian.org>
Last-Update: 2011-01-01
Index: arc-5.21p/arc.1
===================================================================
--- arc.1	2011-01-01 13:26:19.000000000 +0000
+++ arc.1	2011-01-01 13:34:35.000000000 +0000
@@ -137,7 +137,7 @@
 .I Arc
 Version 2 differs from version 1 in that archive entries
 are automatically compressed when they are added to the archive,
-making a separate compression step unecessary.  The nature of the
+making a separate compression step unnecessary.  The nature of the
 compression is indicated by the header version number placed in
 each archive entry, as follows:
 .nf
